.eslintrc.js

```diff
@@ -88,7 +88,7 @@ module.exports = {
   ],
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    ecmaVersion: 2019,
+    ecmaVersion: 2020,
```

Comment:
needed for `BigInt64Array` (available since Node 10.8) in a test. If anyone doesn't feel good updating this for the whole project, I can also just add `BigInt64Array` as an eslint global to just the test file